### PR TITLE
Add logic for otel fields

### PR DESCRIPTION
**Pull request by dbason**
Updated logic for OTEL collector.
We make the following transformations in OTEL
```yaml
processors:
  resource:
    attributes:
    - key: container_image
      action: insert
      from_attribute: container.image.name
    - key: deployment
      action: insert
      from_attribute: k8s.deployment.name
    - key: deployment
      action: insert
      from_attribute: k8s.deployment.name
    - key: pod_name
      action: insert
      from_attribute: k8s.pod.name
    - key: namespace_name
      action: insert
      from_attribute: k8s.namespace.name
    - key: log
      action: delete
  attributes:
    actions:
    - key: filename
      action: insert
      from_attribute: log.file.path
    - key: log.file.path
      action: delete
    - key: message
      action: insert
      from_attribute: log
    - key: log
      action: delete
```

### CODE REVIEW

File: AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java [hunk 1]
```diff
@@ -281,7 +281,16 @@ else if (ingestDocument.hasField("COMM")){
                 if (kubernetes.containsKey("namespace_name")) {
                     namespaceName = ((String)kubernetes.get("namespace_name"));
                 } 
-            }        
+            }
+            if (ingestDocument.hasField("container_image")) {
```
Comment (tybalex) [inline AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:285]: should it be `else if` here? I am considering the case that document has both `kubernetes` and `container_image` field.

Comment (dbason) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:285]: I think container_image would supercede it in that case so I think this logic is OK

Comment (tybalex) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:285]: cool.

**Review by tybalex — APPROVED** [review]
look good.

**Pull request merged**